src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.syfo.client.pdl
+
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import no.nav.syfo.client.azuread.AzureAdV2Client
+import no.nav.syfo.client.httpClientDefault
+import no.nav.syfo.metric.COUNT_CALL_PDL_FAIL
+import no.nav.syfo.metric.COUNT_CALL_PDL_SUCCESS
+import no.nav.syfo.util.*
+import org.slf4j.LoggerFactory
+
+class PdlClient(
+    private val azureAdV2Client: AzureAdV2Client,
+    private val pdlClientId: String,
+    private val pdlUrl: String,
+) {
+    private val httpClient = httpClientDefault()
+
+    suspend fun navn(
+        fnr: String,
```

Comment:
pirk: Endre parameter til personIdentNumber: PersonIdentNumber i stedet for String her og under. Så sikrer man at string må være på fødselsnr-format samt mer konsistent med AdressebeskyttelseClient, KontaktinformasjonClient.